src/App.tsx

```diff
@@ -16,7 +23,16 @@ function App() {
   const { data } = useQuery<IsUserLoggedIn>(IS_LOGGED_IN);
   const me = useMe();
   return !data || !data.isLoggedIn ? (
-    <Login />
+    <Router>
+      <Switch>
+        <Route exact path="/">
```

Comment:
We should fall back to Login route by default if the `/register` path is not met. That way, if someone is on a specific URL (e.g. http://localhost:3000/#/foo/bar), we will still show the login form.